src/web/routes/application/check/get.js

```diff
@@ -38,6 +31,17 @@ const getRowData = (req) => (step) => {
   return Array.isArray(result) ? result.map(applyPathToRow) : applyPathToRow(result)
 }
 
+// a step is navigable if it hasn't defined an isNavigable function.
+const stepIsNavigable = (step, session) => !step.hasOwnProperty('isNavigable') || step.isNavigable(session)
+
+const getLastNavigablePath = (steps, req) => {
```

Comment:
Is there a missing unit test for `getLastNavigablePath`?

Reply:
Added unit test.